pouta_blueprints/models.py

```diff
@@ -214,7 +214,7 @@ class Instance(db.Model):
     id = db.Column(db.String(32), primary_key=True)
     user_id = db.Column(db.String(32), db.ForeignKey('users.id'))
     blueprint_id = db.Column(db.String(32), db.ForeignKey('blueprints.id'))
-    name = db.Column(db.String(64), unique=True)
```

Comment:
We can't drop the uniqueness requirement as long as PVC driver trusts it to be unique